src/queries/models/rewards/generateUserIdModelCreator.ts

```diff
@@ -0,0 +1 @@
+export const generateUserIdModelCreator = (response: any): any => response;
```

Comment:
@abhishek-01k proper types instead of any

Reply:
@corlard3y Can you take a look into this type because you have worked on creating user Id API

Reply:
I already created a create user id API with its own query files, i think this files should be removed cause that's a repetition @abhishek-01k

Reply:
okay then I will remove this file